sky/provision/provisioner.py

```diff
@@ -383,8 +388,9 @@ def wait_for_ssh(cluster_info: provision_common.ClusterInfo,
     def _retry_ssh_thread(ip_ssh_port: Tuple[str, int]):
         ip, ssh_port = ip_ssh_port
         success = False
+        provision_ssh_timeout = skypilot_config.get_nested(('provision_ssh_timeout'), 10)
```

Comment:
Need to make it a tuple?
```suggestion
        provision_ssh_timeout = skypilot_config.get_nested(('provision_ssh_timeout', ), 10)
```